docker/api/network.py

```diff
@@ -0,0 +1,29 @@
+class NetworkApiMixin(object):
+    def networks(self, name=None, partial_id=None):
+        url = self._url("/networks")
+        params = {
+            'name': name,
+            'partial-id': partial_id,
+        }
+        res = self._get(url, params=params)
+        return self._result(res, json=True)
+
+    def create_network(self, name):
+        data = {
+            'name': name,
+            'network_type': '',
+            'options': {},
+        }
+        url = self._url("/networks")
+        res = self._post_json(url, data=data)
+        return self._result(res, json=True)
+
+    def remove_network(self, net_id):
+        url = self._url("/networks/{0}".format(net_id))
```

Comment:
use `self._url('/networks/{0}', resource_id=net_id)` instead to avoid URL tampering